extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -614,8 +616,12 @@ void resetInternal(DataSourceMetadata dataSourceMetadata)
           }
         }
         if (metadataUpdateSuccess) {
-          killTaskGroupForPartitions(JavaCompatUtils.keySet(resetKafkaMetadata.getKafkaPartitions()
-                                                                              .getPartitionOffsetMap()));
+          killTaskGroupForPartitions(
+              JavaCompatUtils.keySet(
```

Comment:
Might as well get rid of JavaCompatUtils along with all the other Java 8 stuff here :)